src/applications/appeals/testing/containers/App.jsx

```diff
@@ -77,6 +77,7 @@ export default function App(props) {
                 labels={chapterTitles.join(';')}
                 total={chapterTitles.length}
                 uswds
+                use-div
```

Comment:
Does `formConfig` need to be passed in `setUpPages`? Just curious what the CodeQL warning is saying.